core/development/ui/Events.ts

```diff
@@ -40,6 +46,8 @@ const SHOP_UPGRADE_SWAPPER = document.getElementById('shop-upgrade-swapper');
 const SHOP_CONTAINER = document.getElementById('shop-container');
 const UPGRADES_CONTAINER = document.getElementById('piece-upgrades-container');
 
+const UNICORN_ATTACK_BUTTON = document.getElementById('unicorn-attack')
```

Comment:
Missing `;`

Reply:
Thats what 8 hours of python a day do to you